docker/compile/Dockerfile.centos7

```diff
@@ -36,6 +36,7 @@ RUN yum -y install \
       make \
       patch \
       python-devel \
+      cppunit-devel \
```

Comment:
Only needed by centos7? What is the error you got?

Reply:
We use centos7, but the following build error occurs.

I think it will be necessary for other os.

```
sh ./scripts/ci/build_release_packages.sh 0.20.3-incubating-rc2 centos7 ./artifacts


ERROR: /root/.cache/bazel/_bazel_root/c481f31d0aff7f9fd86654fb84c9a629/external/org_apache_zookeeper/BUILD.bazel:7:1: Executing genrule @org_apache_zookeeper//:zookeeper-srcs failed (Exit 1): bash failed: error executing command
  (cd /root/.cache/bazel/_bazel_root/c481f31d0aff7f9fd86654fb84c9a629/execroot/org_apache_heron && \
  exec env - \
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
  /bin/bash -c 'source external/bazel_tools/tools/genrule/genrule-setup.sh; export INSTALL_DIR=$(pwd)/bazel-out/k8-opt/bin/external/org_apache_zookeeper
export TMP_DIR=$(mktemp -d -t zookeeper.XXXXX)
export CXXFLAGS=-Wno-error
export CFLAGS=-Wno-error
mkdir -p $TMP_DIR
cp -R $(pwd)/external/org_apache_zookeeper/* $TMP_DIR
cd $TMP_DIR/zookeeper-client/zookeeper-client-c
autoreconf -if
./configure --prefix=$INSTALL_DIR --enable-shared=no
make install
rm -rf $TMP_DIR')
configure: error: Package requirements (cppunit >= 1.10.2) were not met:
No package 'cppunit' found
Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
Alternatively, you may set the environment variables CPPUNIT_CFLAGS
and CPPUNIT_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
libtoolize: putting auxiliary files in `.'.
```

Reply:
kk. Thx.

Let me try. Maybe bazel 3 requires newer things than 0.26.

Reply:
I got the same error. Thx for the fix.